test/infamy/container.py

```diff
@@ -17,6 +19,12 @@ def _find(self, name):
                 return container
         return None
 
+    def _warn(self, msg):
+        """Print a warning message in yellow to stderr."""
+        GUL = "\033[93m"
```

Comment:
Hate to be the party pooper (Go :sweden:! :smile:) ...but I think we should opt for maximum readability, i.e. `s/GUL/YELLOW/`

Reply:
:smirk: